math/mathcore/inc/Fit/FitData.h

```diff
@@ -185,7 +185,7 @@ namespace ROOT {
 
             for (unsigned int i = 0; i < fDim; i++) {
                fCoords[i].resize(fMaxPoints + VectorPadding(fMaxPoints));
-               fCoordsPtr[i] = &fCoords[i].front();
+               fCoordsPtr[i] = fCoords[i].empty() ? NULL : &fCoords[i].front();
```

Comment:
please use nullptr ;)

Reply:
There where already a lot of NULL in those files and I didn't want to introduce inconsistencies by having both NULL and nullptr in the same file. I have made a commit that replaces all NULL with nullptr in the files changed by the PR.